views/faq.handlebars

```diff
@@ -270,6 +270,29 @@
               "makerspace" items to locate the ones that can use your contribution!
             </p>
           </dd>
+          <dt class="btn-link">
+            <h5 class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners">Who are #findthemasks Makers partners?</h5>
+          </dt>
+          <dd class="collapse">
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-1">
+              Our #findthemakers is in partnership with Nation of Makers. Nation of Makers is a national nonprofit 
+              supporting America’s maker organizations through community building, resource sharing, and advocacy, 
+              within the maker movement and beyond. Nation of Makers is supporting maker organizations during the 
+              COVID-19 crisis by connecting them to facilities that need PPE, identifying suppliers (and discounts) 
+              for maker groups fabricating PPE, and identifying funding sources for groups producing PPE. They are 
+              also collating resources for makers responding to COVID-19.
+            </p>
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-2">
+              #findthemasks works with Shield Net, a group that matches organizations in need of medical face 
+              shields with manufacturers that produce them. They have amassed a wide network of university makerspaces 
+              and commercial manufacturers who are making face shields based on their open-source design.
+            </p>
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-3">
+             If you are a makerspace of any size, join their face shield network here. 
```

Comment:
if you want these to all be on their own separate lines you'll need them to be in separate <p> tags

Reply:
Can I just cheat and use <br> ?

Reply:
`<p>` is better for the translators